lib/api/BackbeatAPI.js

```diff
@@ -378,8 +378,8 @@ class BackbeatAPI {
      * @param {Function} cb = callback(error, data)
      * @returns {undefined}
      */
-    monitoringHandler(details, cb) {
-        const promMetrics = ZenkoMetrics.asPrometheus();
+    async monitoringHandler(details, cb) {
+        const promMetrics = await ZenkoMetrics.asPrometheus();
```

Comment:
could have been written using promise's callback api: `ZenkoMetrics.asPrometheus().then(...)`, it looks kind of contradictory to have an async method being provided with a completion callback...